src/Analyzers/MSTest.Analyzers/xlf/Resources.tr.xlf

```diff
@@ -549,7 +549,7 @@ Bu yöntemleri bildiren tipin ayrıca aşağıdaki kurallara uyması gerekir:
       </trans-unit>
       <trans-unit id="TestClassShouldBeValidMessageFormat">
         <source>Test class '{0}' should be valid</source>
-        <target state="translated">'{0}' test sınıfı geçerli olmalıdır</target>
+        <target state="translated">'{0}' test türü geçerli olmalıdır</target>
```

Comment:
@Evangelink It seems like there are unintended changes in already-translated resources?